extension/content_script.js

```diff
@@ -264,10 +263,9 @@ function highlight(tracker) {
     	length: end - start
 	}], {
 		// "trackerClass" is for finding current tracker
-		className: sentenceClass
+		className: sentenceClass+" "+sentenceStyle
```

Comment:
This adds both classes at once

Reply:
oo nice find! I didn't know you can do this